docs/src/index.md

```diff
@@ -4,14 +4,55 @@
 CurrentModule = InfrastructureSystems
 ```
 
-**The Documentation is still under construction, some sections are unfinished. The available methods have autogenerated documentation in the internal API section**
+### Overview
 
-## Overview
-
-`InfrastructureSystems.jl` is a [`Julia`](http://www.julialang.org) package that utilities for
-the packages in NREL's [SIIP Initiative](https://github.com/NREL-SIIP). This package is meant for package development. It is used primarily by
+`InfrastructureSystems.jl` is a [`Julia`](http://www.julialang.org) ppackage that provides
+data management services and common utility software for the packages in
+NREL's [SIIP Initiative](https://github.com/NREL-SIIP). This package is meant
+for module development. It is used primarily by
 [PowerSystems.jl](https://github.com/NREL-SIIP/PowerSystems.jl) and
 [PowerSimulations.jl](https://github.com/NREL-SIIP/PowerSimulations.jl) but is
-written to be extensible for other kinds of applications as well within.
+written to be extensible for other kinds of infrastructure models.
 
 This document describes how to integrate it with other packages.
+
+### Installation
+
+The latest stable release of `InfrastructureSystems.jl` can be installed using the Julia
+package manager with
+
+```Julia
+] add InfrastructureSystems
+```
+
+For the current development version, "checkout" this package with
+
+```Julia
+] add InfrastructureSystems#master
+```
+
+### Usage
+
+`InfrastructureSystems.jl` does not export any method or struct by design. Please refer to
```

Comment:
The style guide reference doesn't seem relevant in the `Usage` section.  The first sentence seems to lack context.  I would leave it out, but I'll leave it to you.